protocol/app/app.go

```diff
@@ -245,6 +248,7 @@ type App struct {
 	ParamsKeeper     paramskeeper.Keeper
 	// IBC Keeper must be a pointer in the app, so we can SetRouter on it correctly
 	IBCKeeper             *ibckeeper.Keeper
+	ICAControllerKeeper   icacontrollerkeeper.Keeper
```

Comment:
Why are we adding `ICAControllerKeeper`, for my understanding? For supporting interchain accounts we specifically wanted to avoid the controllerkeeper ([context](https://www.notion.so/dydx/Supporting-Interchain-Accounts-Liquid-Staking-33e7b09f0bcf44a5b3d1d6931c58d99c?pvs=4#a83746557b00456592390583a6641cdc)).

Reply:
The ICAControllerKeeper is still disabled since `Enabled` is [false in the upgrade](https://github.com/dydxprotocol/v4-chain/blob/3e2c518826ea9d76bb80bdb50f9d0ee955d354bd/protocol/app/upgrades/v3.0.0/upgrade.go#L141) (note the `Enabled` field is taking on the default value).

The [simulation](https://github.com/cosmos/ibc-go/blob/2551dea41cd3c512845007ca895c8402afa9b79f/modules/apps/27-interchain-accounts/simulation/proposals.go#L23) in IBC v8 expects that the controller module to exist.

Reply:
Instead of going with this approach, is it possible to simply exclude ICA module from simulation testing? We've found that simulation testing is not very high signal for the dYdX chain, let alone a third-party module that is tested  elsewhere. 

We specifically prefer to explicitly disable (in code, instead of through gov-updated params) the controller sub-module altogether to reduce surface of risk. Simulation testing only does not justify altering the original approach, IMO. Lmk if the above is possible and happy to huddle!

Reply:
I swapped to exclude the `interchainaccounts` module from simulation as suggested.